server/apps/main/tests/test_views.py

```diff
@@ -672,8 +672,63 @@ def test_list_public_exp_pagination(self):
         response = c.get("/main/public_experiences/?page=2")
 
         # The second page should have 6 experiences
-        print(f'length of experiences: {len(response.context["experiences"])}')
         assert len(response.context["experiences"]) == 6
 
         # The first item on the second page should be the 11th item overall
         assert response.context["experiences"][0].number == 11
+            
+
+    # Test pagination in my_stories, uses separate tests for page 1 / page 2 to avoid cassette issues
+    @vcr.use_cassette(
+            "server/apps/main/tests/fixtures/pag_mystories.yaml",
+            record_mode="none",
+            filter_query_parameters=["access_token"],
+            match_on=["path"],
+    )
+    # This casette contanis 64 stories. 56 are public, 8 are private
+    # of the 56 public stories:
+    # 2 are in review
+    # 22 are not reviewed
+    # 25 are approved
+    # 7 are rejected
+    def test_my_stories_pagination_page1(self):
+        c = Client()
+        c.force_login(self.user_b)
+        
+        response = c.get("/main/my_stories/")
+        num_items_per_page = 10
```

Comment:
Are those breaks hardcoded in the cassette or would it work with any number? I.e. if we were to change this to 5 here, would this still work? If it would still work – should we grab the "correct" number from the settings?

Reply:
Good question. I just checked and the pagination is hardcoded in the cassette. I don't really have an idea how to avoid that, but happy to hear suggestions!

Reply:
That makes sense re: the cassettes, I forgot that it needs to fetch and process the data from OH depending on where the data comes from. I guess it would work for the `PublicExperiences` view, but I think it's fine if we don't have the test adjust to any `num_items_per_page` but rather sticks with what we've set as the default!

Reply:
yes exactly, ok, let's stick with it for now.